packages/web/server/index.ts

```diff
@@ -101,12 +101,20 @@ function wwwRedirect(req: express.Request, res: express.Response, nextAction: ()
     res.redirect('/papers/cLabs_CBDC_Velocity_v3.pdf')
   })
 
+  server.get('/papers/cbdc-velocity/spanish', (_, res) => {
+    res.redirect('/papers/cLabs_CBDC_Velocity_Spanish.pdf')
```

Comment:
super NIT: would probably be nicest if all the papers used same word spacing (dashes)